_includes/hero.html

```diff
@@ -20,33 +20,26 @@
                     </a>
                 -->
 
-                <a href="{{ page.socials.livestream }}">
-                    <button id="livestream" type="button" class="btn btn-outline-primary"><i class="fas fa-video"></i> Watch Livestream</button>
+                <a href="{{ page.socials.youtube }}">
+                    <button id="livestream" type="button" class="btn btn-outline-primary"><i class="fas fa-video"></i> Workshops, Talks & Ceremonies</button>
                 </a>
 
-                <a href="{{ page.socials.platform }}">
-                    <button id="livestream" type="button" class="btn btn-outline-primary"><i class="fas fa-users"></i> Visit Platform</button>
+                <a href="{{ page.socials.devpost }}">
+                    <button id="livestream" type="button" class="btn btn-outline-primary"><i class="fas fa-code"></i> Submissions & Winners</button>
                 </a>
 
                 <div id="register">
                     <a href="{{ page.socials.discord }}">
-                        <button type="button" class="btn btn-primary"><i class="fab fa-discord"></i> 1. Join Discord</button>
+                        <button type="button" class="btn btn-primary"><i class="fab fa-discord"></i> Join Discord</button>
                     </a>
-                    <a href="{{ page.socials.devpost }}">
-                        <button type="button" class="btn btn-primary"><i class="fas fa-code"></i> 2. Register on Devpost</button>
+                    <a href="{{ page.socials.donate }}">
+                        <button type="button" class="btn btn-primary"><i class="fas fa-money-check-alt"></i> Donate to WHO Fund</button>
                     </a>
                     <a href="{{ page.socials.schedule }}">
-                        <button type="button" class="btn btn-primary"><i class="fas fa-calendar"></i> 3. View Schedule</button>
+                        <button type="button" class="btn btn-primary"><i class="fas fa-calendar"></i> View Schedule</button>
```

Comment:
does the schedule even matter at this point?

Reply:
might be useful for seeing what workshops there were?

Reply:
we can change this later if needed, i'm going to get this merged for now

Reply:
Removed